OpenRA.Game/GameInformation.cs

```diff
@@ -34,7 +34,7 @@ public class GameInformation
 
 		/// <summary>Gets the game's duration, from the time the game started until the replay recording stopped.</summary>
 		public TimeSpan Duration { get { return EndTimeUtc > StartTimeUtc ? EndTimeUtc - StartTimeUtc : TimeSpan.Zero; } }
-		public IList<Player> Players { get; private set; }
+		public IList<Player> Players { get; set; }
```

Comment:
What is this required for? Surely a `private` here just prevents you from assigning a new list, not adding elements to the list?

Reply:
Yeah for a while I wasn't able to set players (probably because of what you're saying: setting the list instead of adding), and it doesn't appear to be necessary anymore. Dropped.